config/routes.rb

```diff
@@ -19,6 +19,5 @@
   delete "/sign_out" => "clearance/sessions#destroy", as: "sign_out"
   get "/sign_up" => "clearance/users#new", as: "sign_up"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root to: 'home#index'
```

Comment:
By removing the home# index, the project automatically enters the Rails page. It doesn't enters the project home page.

![imagen](https://user-images.githubusercontent.com/12243331/104506237-ba1a0200-55aa-11eb-8a43-6faf44a1a5fa.png)

Reply:
As I explained in the respective commit message, there were 2 different root routes declared in the file, so that was the thing causing error, I removed the second declaration in order to work, the current state of the file is this one:
![code](https://user-images.githubusercontent.com/39472121/104622652-26523f80-5657-11eb-82e1-996e2618e0cb.png)

And if I run the app on the root route this appears if I'm logged in:
![Screen Shot 2021-01-14 at 10 58 47](https://user-images.githubusercontent.com/39472121/104623123-aa0c2c00-5657-11eb-823f-280d6a4e7998.png)

But if I'm not logged, this one appears: 
![Screen Shot 2021-01-14 at 10 59 07](https://user-images.githubusercontent.com/39472121/104623163-b42e2a80-5657-11eb-9951-cb095049e25f.png)

According to Git Blame, Karina changed the first root route, all I did was removing the second one to resolve conflicts with the Rail Router

Reply:
If I put that root route, I get the tribo home page.

`root to: 'home#index'`

![imagen](https://user-images.githubusercontent.com/12243331/104642928-2cedb080-5671-11eb-8543-8a495f5f22d7.png)

Reply:
Will do

Reply:
Change done!

Reply:
Verified.